src/arguments.h

```diff
@@ -166,16 +172,15 @@ class Arguments {
 
     static long long hash(const char* arg);
     static Output detectOutputFormat(const char* file);
-    static long parseUnits(const char* str, const Multiplier* multipliers);
     static int parseTimeout(const char* str);
 
   public:
     Action _action;
     Counter _counter;
     const char* _event;
+    std::vector<const char*> _trace;
```

Comment:
Well... For a long time, I tried to avoid STL classes in `Arguments`, hoping to get rid of libstdc++ completely some day. That also helped to keep all content of the class static (with `_buf` being the only dynamic field that is allocated just once).

I think I'm not that categorical anymore, but nonetheless, what I can't accept is nonuniformity in argument handling. Let's agree on how to store lists and stick with that approach for all similar arguments. Currently, we handle `include`/`exclude` lists differently - see `Arguments::appendToEmbeddedList`. I admit it's ingenious but hacky :) I'm open to changing it, but my point is that `include`, `exclude` and `trace` lists should follow the same pattern.

Reply:
My personal preference would be changing all of them to use std::vector :)

What do you think? But also, maybe it's not a problem we need to settle in this PR, I changed `_trace` to use the common pattern: 388115571924afbd3b3927ae8a31aed5f0140c07